contents/blog/the-posthog-array-1-42-0.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2022-11-30
+title: "Array 1.42.0: Making recordings easier to share"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available! You can also subscribe to [our Hogmail newsletter](/newsletter)!
+
+## 1.42.0 update guide for self-hosted users
+
+Self-hosted users must run async migrations 0005-0007 before updating to 1.42.0. 
+
+If you haven't run async migrations 0005-0007 yet, or if they aren't available on your current version,  we recommend upgrading to 1.41.4 first, _then_ running the async migrations, then upgrading to 1.42.0.
+
+## PostHog 1.42.0 release notes
+
+**Release highlights:**
+
+- [New: Area chart visualization](#new-area-chart-visualization)
+- [New: Path breakdown normalisations](#new-path-breakdown-normalisations)
+- [New: Share recordings more easily](#new-share-recordings-more-easily)
+- [New: Vote on our roadmap](#new-vote-on-our-roadmap)
+- [Boring legal bit: Policy updates](#boring-legal-bit-policy-updates)
+
+### New: Area chart visualization
+
+![area chart visualization in posthog](../images/blog/array/1-42-0-area-charts.gif)
+
+New team member Thomas Obermueller has got off to a great start by delivering the much-requested area chart visualization for PostHog insights. Now, if a normal line chart isn't  enough and a regular bar chart won't satisfy then you have an exciting third option.
+
+To [quote Thomas](https://github.com/PostHog/posthog/pull/12869) himself: "_The main use case for this is when you want to see the composition of a metric, while also having the total value as a reference e.g. breakdown of revenue by customer over time._"
+
+### New: Path breakdown normalisations
+
+![path breakdown normalization](../images/blog/array/1-42-0-paths.gif)
```

Comment:
I'm forcing myself to close my laptop and stop working for today, so sorry to not fix this directly

This change doesn't affect the Paths insight. It's when breaking down an insight by $pathname or $current_url

@joethreepwood

Reply:
probably want one of the gifs from here https://github.com/PostHog/posthog/pull/12808

Reply:
@joethreepwood I updated the gif and text.

> "so you'll see there's only one place like /home."

Very much on board with this joke btw

Reply:
Thanks, Paul! <3

Reply:
The confusion highlights a good feature request though https://github.com/PostHog/posthog/issues/13002